Jenkinsfile

```diff
@@ -1,65 +1,135 @@
 #!groovy
 @Library('amf-jenkins-library') _
 
+def SLACK_CHANNEL = '#amf-jenkins'
+def PRODUCT_NAME = "scala-common"
+def lastStage = ""
+def color = '#FF8C00'
+def headerFlavour = "WARNING"
+
 pipeline {
-  agent {
-    dockerfile true
-  }
-  environment {
-    NEXUS = credentials('exchange-nexus')
-    GITHUB_ORG = 'aml-org'
-    GITHUB_REPO = 'scala-common'
-  }
-  stages {
-    stage('Test') {
-      steps {
-        sh 'sbt clean coverage test coverageReport'
-      }
+    options {
+        timeout(time: 30, unit: 'MINUTES')
+        ansiColor('xterm')
     }
-    stage('Coverage') {
-      when {
-        anyOf {
-          branch 'master'
-          branch 'sonar-onboard'
+    agent {
+        dockerfile {
+            filename 'Dockerfile'
```

Comment:
Do you want to use our custom base image here ? This will not work as the Dockerfile doesn't use the base image.